api/src/libs/healthCheck.ts

```diff
@@ -0,0 +1,46 @@
+import { LogWriterType } from "@libs/logWriter";
+import axios, { AxiosError, AxiosResponse } from "axios";
+
+type Status = "PASS" | "FAIL" | "ERROR";
+type StatusResult = Record<string, Status>;
+
+const healthCheckEndPoints: Record<string, string> = {
+  self: "self",
+  elevator: "dynamics/elevator",
+  fireSafety: "dynamics/fire-safety",
+  housing: "dynamics/housing",
+  dynamicsLicenseStatus: "dynamics/license-status",
+  webserviceLicenseStatus: "webservice/license-status",
+  webserviceFormation: "webservice/formation",
+};
+
+const url =
+  process.env.STAGE === "prod"
+    ? "https://api.navigator.business.nj.gov"
+    : "https://dev.api.navigator.business.nj.gov";
```

Comment:
[sand] Let's create environment variables for these, if we don't already have them. That way it's easier for us to change them in the future.

Reply:
Also, since we're not appending to this string, it should be called `domain` or `urlBase`, since it won't be the final full resource locator.